source/faucet/main.d

```diff
@@ -238,14 +242,14 @@ public class Faucet : FaucetAPI
         static assert (isInputRange!UR);
 
         return utxo_rng
-            .filter!(tup => tup.value.output.value >= Amount(count))
+            .filter!(tup => tup.value.output.value >= minInputValuePerOutput * count)
             .map!(tup => TxBuilder(tup.value.output, tup.key))
-            .map!(txb => txb.split(
+            .map!(txb => txb.unlockSigner(&this.keyUnlocker).split(
```

Comment:
I think we should be a bit smarter here (attaching to multiple UTXOs) but for now this works.